packages/calling-stateful-client/src/CallFeatureStreamUtils.ts

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/* @conditional-compile-remove(together-mode) */
+import { CreateViewOptions, VideoStreamRenderer } from '@azure/communication-calling';
+/* @conditional-compile-remove(together-mode) */
+import { CallContext } from './CallContext';
+/* @conditional-compile-remove(together-mode) */
+import {
+  CallFeatureStreamName,
+  CallFeatureStreamState,
+  CreateViewResult,
+  VideoStreamRendererViewState
+} from './index-public';
+/* @conditional-compile-remove(together-mode) */
+import { InternalCallContext } from './InternalCallContext';
+/* @conditional-compile-remove(together-mode) */
+import { _logStreamEvent } from './StreamUtilsLogging';
+/* @conditional-compile-remove(together-mode) */
+import { EventNames } from './Logger';
+/* @conditional-compile-remove(together-mode) */
+import { convertFromSDKToDeclarativeVideoStreamRendererView } from './Converter';
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * @private
+ *
+ */
+export function createCallFeatureView(
```

Comment:
not blocking this PR as I understand there was guidance to make this an isolated function, but this really should be integrated with the existing createCallView functions.
Let's leave as is today, confirm there's no issues when testing in selfhost etc. then when we get close to release time add a work item to refactor this into the existing stream utils.